src/lib/styles/global/link.scss

```diff
@@ -13,4 +13,48 @@ a {
   &:hover {
     color: var(--value-color);
   }
+
+  &.button {
```

Comment:
Just thinking out loud, would it make sense to create a button `mixin` to share some paddings and colors?

Reply:
Yes, I agree. Let me do it.

Reply:
Thank you!

Reply:
done!